contrib/build-flatpak.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+export LC_ALL=C
+set -e
+SECONDS=0
+
+repo_root="$(realpath $(dirname "$0")/..)"
+mkdir -p "$repo_root/build"
+cd "$repo_root/build"
+
+echo $repo_root
```

Comment:
```suggestion
echo "$repo_root"
```
nit (shellcheck warn)